frontend/app/src/components/home/Rules.svelte

```diff
@@ -40,6 +48,19 @@
         maxlength={MAX_RULES_LENGTH}
         rows={8}
         placeholder={interpolateLevel("rules.placeholder", level, true)} />
+    {#if editing && rules.enabled}
+        <Toggle
```

Comment:
Not sure I get the idea here? Are we allowing the user to _decide_ whether the change they have made to the rules should represent a new version of the rules which much therefore be accepted by people again?